images/receiveGpgKeys.sh

```diff
@@ -6,19 +6,19 @@
 
 set -ex
 
-KEYS="$@"
+KEY_IDS="$@"
 
-if [ "$KEYS" == "" ]; then
+if [ "$KEY_IDS" == "" ]; then
```

Comment:
What this script gets as input are really ids of the actual public keys that we are trying to receive.